db/models.py

```diff
@@ -41,12 +50,60 @@ def __str__(self) -> str:
 
 class MovieSession(models.Model):
     show_time = models.DateTimeField()
-    cinema_hall = models.ForeignKey(
-        to=CinemaHall, on_delete=models.CASCADE, related_name="movie_sessions"
-    )
-    movie = models.ForeignKey(
-        to=Movie, on_delete=models.CASCADE, related_name="movie_sessions"
-    )
+    cinema_hall = models.ForeignKey(CinemaHall, on_delete=models.CASCADE)
+    movie = models.ForeignKey(Movie, on_delete=models.CASCADE)
+
+    def __str__(self) -> str:
+        return (str(self.movie) + " "
+                + self.show_time.strftime("%Y-%m-%d %H:%M:%S"))
+
+
+class Order(models.Model):
+    created_at = models.DateTimeField(auto_now_add=True)
+    user = models.ForeignKey(settings.AUTH_USER_MODEL,
+                             on_delete=models.CASCADE)
+
+    def __str__(self) -> str:
+        return self.created_at.strftime("%Y-%m-%d %H:%M:%S")
+
+    class Meta:
+        ordering = ["-created_at"]
+
+
+class Ticket(models.Model):
+    row = models.IntegerField()
+    seat = models.IntegerField()
+    movie_session = models.ForeignKey(MovieSession, on_delete=models.CASCADE)
+    order = models.ForeignKey(Order, on_delete=models.CASCADE)
 
     def __str__(self) -> str:
-        return f"{self.movie.title} {str(self.show_time)}"
+        return f"{self.movie_session} (row: {self.row}, seat: {self.seat})"
+
+    def clean(self) -> None:
+        if not (1 <= self.seat <= self.movie_session.cinema_hall.seats_in_row):
+            quant_of_seat = self.movie_session.cinema_hall.seats_in_row
+            raise ValidationError(
+                {"seat": f"seat number must be in available range: "
+                         f"(1, seats_in_row): (1, {quant_of_seat})"}
+            )
+        if not (1 <= self.row <= self.movie_session.cinema_hall.rows):
+            quant_of_row = self.movie_session.cinema_hall.rows
+            raise ValidationError(
+                {"row": f"row number must be in available range: "
+                        f"(1, rows): (1, {quant_of_row})"}
+            )
+
+    def save(self, *args, **kwargs) -> None:
+        self.full_clean()
+        return super().save(*args, **kwargs)
+
+    class Meta:
+        # unique_together = ["seat", "trip"]
+        constraints = [
+            models.UniqueConstraint(fields=["seat", "row", "movie_session"],
+                                    name="unick_ticket")
```

Comment:
constraint name should be more descriptive, it needs to tell exactly what it does e.g. `row_seat_per_session_uniq`